.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,41 @@
+name: PR
+
+on:
+    pull_request:
+      types: [opened, reopened, synchronize]
+
+jobs:
+    soundness:
+        name: Soundness
+        uses: apple/swift-nio/.github/workflows/soundness.yml@main
+        with:
+            # These are set to false to stage this in with the old CI.
+            # A follow-up PR will cut them over.
+            api_breakage_check_enabled: false
+            broken_symlink_check_enabled: false
+            docs_check_enabled: false
+            format_check_enabled: false
+            license_header_check_enabled: false
+            license_header_check_project_name: "SwiftASN1"
+            shell_check_enabled: false
+            unacceptable_language_check_enabled: false
+    unit-tests:
+        name: Unit tests
+        uses: apple/swift-nio/.github/workflows/unit_tests.yml@main
+        with:
+            linux_5_8_enabled: false
+            linux_5_9_arguments_override: "--explicit-target-dependency-import-check error"
+            linux_5_10_arguments_override: "--explicit-target-dependency-import-check error"
```

Comment:
We are we not passing `--warnings-as-errros` here?

Reply:
Good catch, that should be there